assembl/static2/js/app/components/common/largeTextParagraph.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Grid } from 'react-bootstrap';
+import Header from './header';
+
+const LargeTextParagraph = ({ debateData, headerTitle, text }) => {
+  return (
+    <div className="large-text-paragraph">
+      <Header title={headerTitle} imgUrl={debateData.headerBackgroundUrl} />
+      <Grid fluid>
```

Comment:
Is Grid really necessary here? I think Grid should only be one time on a page, and should include Rows in it.

Reply:
Probably unnecessary indeed

Reply:
Look at the synthesis page. The only `<Grid fluid>` needed is the container for intro, tree and conlusion, this is to have a thin margin for mobile.